src/Plugins/RabbitMQ/Factory/RabbitMqConnectionFactory.cs

```diff
@@ -86,7 +73,40 @@ public IModel CreateChannel(string hostName, string username, string password, s
                     }
                 });
 
-            return connection.Value.CreateModel();
+            var model = connection.Value.CreateModel();
+
+            var argsObj = new CreateChannelArguments(hostName, password, username, virtualHost, useSSL, portNumber);
+
+            model.CallbackException += (connection, args) => OnException(args, key, argsObj);
+            model.ModelShutdown += (connection, args) => OnShutdown(args, key, argsObj);
+
+            return model;
+        }
+
+        private void OnShutdown(ShutdownEventArgs args, string key, CreateChannelArguments createChannelArguments)
+        {
+            _logger.LogError($"RabbitMQ connection shutdown attempting to reconnect.", args);
```

Comment:
Please use structured logging and assign an event ID to this log entry.  

Looking at other components, we can likely start with `10000`